resources/logging/profile-evaluation.yaml

```diff
@@ -17,3 +17,12 @@ fluent-bit:
     requests:
       cpu: 20m
       memory: 40Mi
+
+logui:
+  resources: 
+    requests:
+      cpu: 1m
+      memory: 4Mi
+    limits:
+      cpu: 1m
```

Comment:
Look what few page reloads have done. I think `3m` value would be better.
![image](https://user-images.githubusercontent.com/22825433/100994872-5b974800-3557-11eb-99d9-ccd1c9d588cf.png)